pkgs/by-name/te/tegola/package.nix

```
@@ -0,0 +1,52 @@
+{ lib, buildGoModule, buildNpmPackage, fetchFromGitHub }:
+let
+  version = "0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "go-spatial";
+    repo = "tegola";
+    rev = "v${version}";
+    sha256 = "sha256-Jlpw3JaU5+DO7Z5qruEMoLRf95cPGd9Z+MeDGSgbMjc=";
+  };
+
+  frontend = buildNpmPackage {
+    pname = "tegola-ui";
+    inherit version;
+
+    src = "${src}/ui";
+
+    npmDepsHash = "sha256-rhUdWt1X5/F0uvT8gI1T9ei6Y+HK1tKj2fuTKlMAwJk=";
+
+    postPatch = ''
+      cp ${./package-lock.json} package-lock.json
```

What about 

```suggestion
      cp ${src}/ui/package-lock.json package-lock.json
```

+ remove `package-lock.json` file from nixpkgs package.

---

I've removed package-lock.json and postPatch, as it turned out it's not needed.